docs/workflow/requirements/windows-requirements.md

```diff
@@ -1,113 +1,117 @@
 
 The runtime repository requires long paths to be enabled. Follow [the instructions provided here](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation#enable-long-paths-in-windows-10-version-1607-and-later) to enable that feature.
 
-If using Git for Windows you might need to also configure long paths there. Using an admin terminal simply type:
+If using Git for Windows you might need to also configure long paths there. Using an administrator terminal simply type:
+
 ```cmd
 git config --system core.longpaths true
 ```
 
-## Visual Studio
+### Visual Studio
 
-- Install [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). The Community edition is available free of charge. Visual Studio 2022 17.3 or later is required.
+Install [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). The Community edition is available free of charge. Visual Studio 2022 17.3 or later is required. Note that Visual Studio and the development tools described below are required, regardless of whether you plan to use the IDE or not. The installation process goes as follows:
 
-Visual Studio 2022 installation process:
-- It's recommended to use **Workloads** installation approach. The following are the minimum requirements:
-  - **.NET Desktop Development** with all default components,
-  - **Desktop Development with C++** with all default components.
-- The build tools (CMake, Ninja and Python) can be downloaded and installed separately (see detailed instructions in the [section below](#build-tools)) or by selecting the following **Individual Components**:
-  - **C++ CMake tools for Windows** (includes Ninja, but might not work on ARM64 machines),
-  - **Python 3 64-bit** (3.7.4 or newer).
-- To build for Arm32 or Arm64, make sure that you have the right architecture-specific compilers installed. In the **Individual components** window, in the **Compilers, build tools, and runtimes** section:
-  - For Arm32, check the box for **MSVC v143 - VS 2022 C++ ARM build tools (Latest)** (v14.31 or newer),
-  - For Arm64, check the box for **MSVC v143 - VS 2022 C++ ARM64 build tools (Latest)** (v14.31 or newer).
-- To build the tests, you will need some additional components:
-  - **Windows 10 SDK (10.0.19041)** or newer. This component is installed by default as a part of **Desktop Development with C++** workload.
-  - **C++/CLI support for v142 build tools (Latest)** (v14.23 or newer).
+* It's recommended to use **Workloads** installation approach. The following are the minimum requirements:
+  * **.NET Desktop Development** with all default components,
+  * **Desktop Development with C++** with all default components.
+* The build tools (CMake, Ninja and Python) can be downloaded and installed separately (see detailed instructions in the [section below](#build-tools)) or by selecting the following **Individual Components**:
+  * **C++ CMake tools for Windows** (includes Ninja, but might not work on ARM64 machines),
```

Comment:
```suggestion
```
This is not necessary. It is included by default in `Desktop Development with C++`